app/src/main/java/com/ataulm/artcollector/gallery/ui/GalleryAdapter.kt

```diff
@@ -32,9 +35,9 @@ internal class GalleryAdapter constructor(
     ) : RecyclerView.ViewHolder(view) {
 
         fun bind(item: UiPainting) {
-            itemView.setOnClickListener { item.onClickPainting(adapterPosition) }
+            itemView.setOnClickListener { item.onClickPainting(itemView.imageView) }
```

Comment:
we pass the imageView for the sharedElement transition. We trust that no one that shouldn't will keep a reference to it 😬 

It was weird passing the adapter position before. It would go back to the ViewModel, then to the activity and the activity would try to find the view from the recyclerview's layoutmanager.

This passes the view to the VM. In my mind, I'd always considered the VM as something that shouldn't really have any android references to it, so that it could be platform agnostic - but since it's extending the _android_ ViewModel, it's kind of pointless 🤷‍♂️